cluster-provision/k8s/1.30/extra-pre-pull-images

```diff
@@ -2,10 +2,9 @@ quay.io/kubevirtci/install-cni:1.15.0
 quay.io/kubevirtci/operator:1.15.0
 quay.io/kubevirtci/pilot:1.15.0
 quay.io/kubevirtci/proxyv2:1.15.0
-quay.io/calico/cni:v3.18.0
-quay.io/calico/kube-controllers:v3.18.0
-quay.io/calico/node:v3.18.0
-quay.io/calico/pod2daemon-flexvol:v3.18.0
+quay.io/calico/cni:v3.27.2
+quay.io/calico/kube-controllers:v3.27.2
+quay.io/calico/node:v3.27.2
```

Comment:
Could the bump of calico be split out to a different PR? is this needed for the s390x provider?

Reply:
The calico bump should be handled in a separate PR - as we will need to do this for all providers to keep it consistent.

Reply:
ok. I'll start a separate PR for calico bump. 
Just to confirm my understanding - we keep (or try to keep)  same version of calico across all provider versions and update it across provider versions when necessary. Isn't it?

Reply:
Yes unless there is a specific reason to have an older version of calico on a certain provider but I don't think there is any reason in this case

Reply:
Done. create separate PR https://github.com/kubevirt/kubevirtci/pull/1281/files